### PR TITLE
A Pull Request On: To Do list: interactive list

### PR DESCRIPTION
**Pull request by atok624**
In This Pull Request, I did the following:
- Added a new JavaScript file and imported it as a module:
it will contain methods related to the status updates (completed: true/false).
- Added event listener to the checkbox (change).
- Updated items object's value for completed key upon user actions.
- Implemented a function for the "Clear all completed" button (use filter() method).
- Stored the updated array of items in local storage, so the user gets the correct list of values after the page reloads (which means that any changes made to the list should be preserved).

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,39 +1,151 @@
...
+
+const clearCompletedTasks = () => {
+  tasks = tasks.filter((task) => !task.completed);
+  saveTasks(tasks);
+  renderTaskList();
+};
+
+const handleDragStart = (event) => {
+  event.target.classList.add('dragging');
+};
```
Comment (TadesseDev) [inline src/index.js]: - [Optional] Impressive job @atok624, implementing the drag and drop functionality. Though it's possible to drag and arrange items on your website, your functionality stops working once I rearrange items and throws exceptions. Such cases will cause bad user experiences. I suggest implementing the functionality if possible or removing it till it is completely functional. 🙌 

https://user-images.githubusercontent.com/69077061/221162482-69f12cca-82f9-4739-93a0-6b42d98d5aa4.mp4

**Review by TadesseDev — APPROVED** [review]
### Hi @atok624 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [ ]  Good commit history. ✅
 - [ ]  Good PR title/description. ✅
 - [ ]  All linters passed ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (atok624) [reply src/index.js]: Okay 
Thank you
I will surely work on that

Comment (TadesseDev) [reply src/index.js]: perfect 👌

**Pull request merged**